uktrade/data-hub-frontend
# feature/CLS2-490-edit-due-date-approaching-settings

## Description of change

- Added a new form for editing due date approaching reminder settings
- Added a new resource for loading the due date approaching settings
- Added an Edit button to the reminder screen to link to the new form

## Test instructions

Go to http://localhost:3000/reminders/settings?my_tasks_due_date_approaching=true and click the Edit button. You will be taken to the edit settings page.

You will be able to change the email settings and see those changes displayed on the settings page
## Screenshots

### Before

N/A

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/0a28c8d8-d93a-40d9-9b5e-0af536bc0c33)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
